test/algorithms/ground_state_solvers/test_swaprz.py

```diff
@@ -44,7 +44,8 @@ def setUp(self):
     def test_excitation_preserving(self):
```

Comment:
Since this test was originally done as a unit test for the SwapRZ variational form as it was in Aqua, and since its moved to Terra where it ought to be unit tested there, It maybe that we can think about getting rid of this later. Its more an integration/system level test here in this context to make sure things work as expected when using it